master/usage/troubleshooting/logging.md

```diff
@@ -34,57 +34,50 @@ See the following sub-sections for details on configuring the log level for
 each calico-node component.
 
 ### Bird/Bird6
-Bird and Bird6 are used for distributing IPv4 and IPv6 routes between Calico
-enabled hosts.
 
-Directory | Default level
---- | ---
-`bird` and `bird6` | `info`
+Bird and Bird6 are used for distributing IPv4 and IPv6 routes between Calico
+enabled hosts.  The logs are output in the `bird` abd `bird6` sub-directories
+of the calico/node logging directory.
 
-Use the `calicoctl config bgp loglevel` command on any host to change the
-log level across all BGP nodes, _or_ use the `calicoctl config node bgp loglevel`
-command on a specific host to change the log level for that specific BGP node.
+Use the `calicoctl config loglevel` command on any host to change the
+log level across all Calico nodes, _or_ use the same command with the `--node`
+option to run the command for that specific node.  This command affects the 
+logging level for both Bird/Bird6 and Felix.
 
-Valid log levels are:  none, debug and info.  e.g.
+Valid log levels are:  none, debug, info, warning, error, critical.  For example:
 
-        calicoctl config bgp loglevel none
-        calicoctl config node bgp loglevel debug
+        calicoctl config loglevel error
```

Comment:
since `logLevel` is not a command arg itself it would be nice to put it inside `<...>`, WDYT?